content/docs/dataform-web/how-to-guides/environments-example.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Environments example project
+---
+
+<div className="bp3-callout bp3-icon-info-sign" markdown="1">
+  Environments require <code>@dataform/core >= 1.4.9</code>.
+</div>
+
+## Introduction
+
+This guide will show you how to configure a project with two environments. One environment will be staging, and the other will be production. Typical of this paradigm, staging will be upstream of production, allowing committed code to be tested before being released.
+
+## Creating a project with environments
```

Comment:
Honestly I think this is redundant. You can just link to the other how to guide (setting up a project). Duplicating this here just means more places to have to change it in the future!

Reply:
Yes now that you've said it I definitely agree